services/offline/src/utils/__tests__/render-counter.test.js

```diff
@@ -0,0 +1,46 @@
+import { act, fireEvent, render, screen } from '@testing-library/react'
```

Comment:
This is just verifying that a 'render counter' test utility works as expected; it's used in the integration tests to verify that the state refactor worked (see the last two tests in [integration.test.js](https://github.com/dhis2/app-runtime/pull/901/files#diff-b97f31d5db63f4213c857e2461871a7ddbb6d09fbedd9e9af2eb0f4ea2fa1817))